server/grpc/grpc_web.go

```diff
@@ -11,7 +11,16 @@ import (
 
 // StartGRPCWeb starts a gRPC-Web server on the given address.
 func StartGRPCWeb(grpcSrv *grpc.Server, config config.Config) (*http.Server, error) {
-	wrappedServer := grpcweb.WrapServer(grpcSrv)
+	var options []grpcweb.Option
+	if config.API.EnableUnsafeCORS {
```

Comment:
So the API and GRPCWeb servers are on different ports. I'm wondering if we should actually add a new config `config.GRPCWeb.EnableUnsafeCORS`